submarine-server/server-api/src/main/java/org/apache/submarine/server/api/spec/ExperimentMeta.java

```diff
@@ -51,6 +55,22 @@ public void setName(String name) {
     this.name = name;
   }
 
+  /**
+   * Get the experiment id which is unique within a namespace.
+   * @return experiment id
+   */
+  public String getExpID() {
```

Comment:
Disallow to abridge the name, because it will reduce readability.
```suggestion
  public String getExperimentId() {
```

Reply:
I have fixed it.